flow/scripts/generate_abstract.tcl

```diff
@@ -17,7 +19,7 @@ if {$design_stage >= 4} {
   set_propagated_clock [all_clocks]
 }
 
-puts "Starting generation of abstract views"
+puts $generating_comment
```

Comment:
It seems redundant to print it twice (since we are cleaning things up)

Reply:
I wanted a single concern for this PR, to fix the text.

I don't understand why load_design takes this parameter or under what circumstances it is printed:

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/c72ffd7dcd317c337df2535314289f8d0a593467/flow/scripts/load.tcl#L33

Reply:
The standalone variable is leftover from an old mode where we ran the entire flow in a single invocation of OR (see run_all.tcl).  We haven't used it for years so I doubt it still works.  The theory was that you could save the write/read overhead between steps when running the whole flow.

In practice standalone is never set today so this else won't happen.

Reply:
Ripping out the standalone stuff is a separate concern, not related to this PR.

Should it be done?

Reply:
Its a nice idea so there was always the thought to resuscitate it someday.  I've approved this and we can table the rest for now.